docs/hybrid-agent-beats-receivers.md

```diff
@@ -214,6 +216,24 @@ exporters:
             enabled: true
         mapping:
             mode: bodymap
+
+        retry:
+            enabled: true
+            initial_interval: 1s
+            max_interval: 1m0s
+            max_retries: 5
```

Comment:
To be equal to filebeat, we need the ability to retry indefinitely. This is covered by https://github.com/elastic/beats/issues/47892 so we probably want to reference that.

Reply:
Thanks, I updated the doc to reflect current max_retries values and link to the issue.